decide/src/main/java/group17/LicAnalyzer.java

```diff
@@ -203,7 +203,45 @@ public boolean lic7(InputHandler input) {
         return false;
     }
 
+    /**
+     * This function calculates Launch Interceptor Condition (LIC) number 8
+     * Utilizes helper methods in GeometryUtils.java to calculate the smallest possible radius between 3 points.
+     * The first two points are separated by A_PTS.
+     * The last two points are separated by B_PTS.
```

Comment:
Emir and I have been wondering about this, but do you interpret these kind of assignment as `A_PTS` specifically separating point 1 and 2 and `B_PTS` separating point 2 and 3? 

If yes, could you explain your thought process since we are confliced.

If no, i.e. the points must just be separated by either one respectively, does the current solution find points in both order (`A_PTS then B_PTS`, but also, `B_PTS then A_PTS`?

Reply:
When I read the requirements I intuitively thought it would be 1 A_PTS 2 B_PTS 3, but upon thinking about it I see the requirement is unclear about this detail. Since my interpretation makes sense according to the requirement since the requirement is unclear, I think this pull request is fine. Checking both ways I think is excessive but good.